### PR TITLE
Katello 3.13 release

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

**Comment by jjeffers** [comment]
@jturel @chris1984 At this point, can you review and indicate what's missing or what should be altered? Thank!

File: configs/katello/3.13.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  :katello:
+    :main: true
+    :branch: KATELLO-3.13
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/1.23
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 0.18.0
```
Comment (chris1984) [inline configs/katello/3.13.yaml]: lets change this to 0.19.0

File: configs/katello/3.13.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :puppet-certs:
+    :branch: 6.0.0
```
Comment (chris1984) [inline configs/katello/3.13.yaml]: 6.0.1

File: configs/katello/3.13.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    :repo: https://github.com/theforeman/puppet-certs
+  :katello-host-tools:
+    :branch: 3.5.0
```
Comment (chris1984) [inline configs/katello/3.13.yaml]: 3.5.1

**Comment by chris1984** [comment]
@zjhuntin can this get a merge please?

**Pull request merged**